shared/js/initApp.js

```diff
@@ -14,7 +14,7 @@ window.apps = {
   // Legacy Blockly initialization that was moved here from _blockly.html.haml.
   // Modifies `appOptions` with some default values in `baseOptions`.
   // TODO(dave): Move blockly-specific setup function out of shared and back into dashboard.
-  setupBlockly: function () {
+  setup: function (appOptions) {
```

Comment:
Wonder if this method could maybe benefit from being renamed `initializeApp` just to namespace a bit for `find usages` down the road